src/lib/components/buttons/button/terra-button.component.ts

```diff
@@ -92,11 +95,8 @@ export class TerraButtonComponent
     @Output()
     public outputClicked:EventEmitter<Event> = new EventEmitter<Event>();
 
-    protected currentTooltipPlacement:string;
-
     constructor()
     {
-        this.inputTooltipPlacement = TerraPlacementEnum.TOP;
```

Comment:
es ist zunächst nur deprecated aber der Tooltip wird sich bereits vor dem Löschen des Inputs in einer Majorversion anders verhalten, da das Input keine Wirkung mehr hat.

Reply:
jetzt wird das in der directive direkt auf top gesetzt